### PR TITLE
Email template changes

### PR DESCRIPTION
**Pull request by nmonga91**

### CODE REVIEW

**Comment by nmonga91** [comment]
@bsweger or @jworcestBAH

File: dataactbroker/scripts/setupEmails.py [hunk 1]
```diff
@@ -23,27 +23,27 @@ def setupEmails():
...
 
     #Admin Email
-    template = "This email is to notify you that the following person has requested an account for the USA Spending Data Broker:<br /><br />Name: [REG_NAME]<br /><br />Title:  [REG_TITLE]<br /><br />Agency:  [REG_AGENCY]<br /><br />Email: [REG_EMAIL]<br /><br /><br /><br />To approve or deny this user for access to the Data Broker, please click <a href='[URL]'>here</a>.<br /><br />This action must be taken within 24 hours. <br /><br />Thank you for your prompt attention.<br /><br />The DATA Act Implementation Team<br />DATAPMO@fiscal.treasury.gov"
+    template = "This email is to notify you that the following person has requested an account for the USA Spending Data Broker:<br /><br />Name: [REG_NAME]<br /><br />Title:  [REG_TITEL]<br /><br />Agency:  [REG_AGENCY]<br /><br />Email: [REG_EMAIL]<br /><br /><br /><br />To approve or deny this user for access to the Data Broker, please click <a href='[URL]'>here</a>.<br /><br />This action must be taken within 24 hours. <br /><br />Thank you for your prompt attention.<br /><br />The DATA Act Implementation Team<br />DATABroker@fiscal.treasury.gov"
```
Comment (bsweger) [inline dataactbroker/scripts/setupEmails.py]: There was a recent hotfix for the REG_TITEL misspelling in the e-mail template. Looks like that's not in there yet.

**Comment by bsweger** [comment]
@nmonga91 done reviewing--can you re-merge dev into this branch to make sure the template fix from @jworcestBAH gets in there?

**Comment by nmonga91** [comment]
@bsweger the merge has been completed and please note the additional commit that brings a couple more changes to the email template.

**Comment by bsweger** [comment]
Look good!

**Pull request merged**